factfinder/main.py

```diff
@@ -199,6 +199,10 @@ def aggregate_horizontal(self, client, v, geotype) -> pd.DataFrame:
         census_variables = E_variables + M_variables
         df = self.download_variable(client, census_variables, geotype)
 
+        for i in M_variables:
+            est = i.replace('M', 'E')
+            df.loc[df[est].isnull(), i] = np.nan
+            
```

Comment:
```python
 for i in v.census_variable:
            df.loc[df[f'{i}M'].isnull(), f'{i}E'] = np.nan
```
Can you also provide some explanation here? Ideally we would do all the data cleaning in the same place, instead of having them scattered around

Reply:
we should probably try to move all these into the download function, instead of passing "variables" around, we should just pass v.census_variable as an argument

Reply:
I'll be working on this